src/common/Requirements.ts

```diff
@@ -22,21 +23,27 @@ export interface Requirement {
     logger: Logger;
 }
 
-export interface RequirementResult {
+interface RequirementResult {
     duration: number;
     hasPassed: boolean;
     message: string;
     title: string;
 }
 
-export interface SetupTestResult {
+interface RequirementCheckResult {
     hasMetAllRequirements: boolean;
     tests: RequirementResult[];
 }
 
 export interface RequirementList {
```

Comment:
Looks like we're not using `RequirementList` outside this file so perhaps we should remove `export`

Reply:
`RequirementList` is used by `AndroidEnvironmentRequirements` and `IOSEnvironmentRequirements`.